### ISSUE TITLE
Select: After selecting an item, only the remaining unselected options should be focusable by keyboard navigation.

### ISSUE THREAD

**Issue by gulcinuras** [Hipo/react-ui-toolkit#203]

**Comment by gulcinuras** [comment]
https://user-images.githubusercontent.com/79009325/189623708-4966ea45-2fb0-43d6-a598-ad9d00f4de76.mov
